src/ReactiveUI.Splat.Tests/ReactiveUI.Splat.Tests.csproj

```diff
@@ -17,5 +17,4 @@
   <ItemGroup>
     <ProjectReference Include="..\ReactiveUI\ReactiveUI.csproj" />
   </ItemGroup>
-
 </Project>
```

Comment:
Whitespace change only so probably doesnt need to be here

Reply:
reverted